R/util.R

```diff
@@ -30,3 +30,14 @@ cols <- function(n) {
 round_time <- function(x) {
   round(x, 2L)
 }
+
+
+## make a color transparent
+transp <- function (col, alpha = 0.5) {
+  apply(grDevices::col2rgb(col),
```

Comment:
apply over `grDevices::col2rgb(col / 255)` and then all the duplicated `/255` can go away.

I think you can simplify even further though with

```
col_rgb <- col2rgb(col / 255)
grDevices::rgb(col_rgb[, 1], col_rgb[, 2],col_rgb[, 3], alpha)
```

Reply:
Couple of mistakes in that code o'yours ( ;p ), but new version now reads:
```
transp <- function (col, alpha = 0.5) {
  col_rgb <- col2rgb(col) / 255
  grDevices::rgb(col_rgb[1, ], col_rgb[2, ],col_rgb[3, ], alpha)
}
```